packages/slate/src/interfaces/node.ts

```diff
@@ -197,7 +197,7 @@ export const Node = {
       )
     }
 
-    const newRoot = produce(root, r => {
+    const newRoot = produce({ children: root.children }, r => {
```

Comment:
Since 7.0.0, Immer freezes all properties of the new state (see https://github.com/immerjs/immer/issues/462 for more details). This caused `editor.operations` to be frozen when calling `Editor.fragment(editor, ...)`.

Reply:
Confused by this comment.  Does this PR introduce an issue with `editor.operations` by freezing it if `Editor.fragment` is called?

Reply:
If I recall correctly, in immer 6 the way freezing works has changed. Previously immer (5) would only freeze the new object, starting with 6 it will also recursively freeze all properties. And because of structural sharing, this means that any property of `root` unchanged by the reducer function would also be frozen.